client/js/components/statement/StatementExportModal.vue

```diff
@@ -54,24 +55,45 @@
         <div class="grid grid-cols-5 gap-3 mt-1 mb-5">
           <dp-input
             v-for="(column, key) in docxColumns"
-            :key="key"
             :id="key"
-            type="text"
+            :key="key"
+            v-model="column.title"
             :data-cy="column.dataCy"
             :placeholder="Translator.trans(column.placeholder)"
-            :width="column.width"
-            v-model="column.title" />
+            type="text"
+            :width="column.width" />
         </div>
+        <template v-if="this.active === 'zip' || isSingleStatementExport">
+          <h3
```

Comment:
First i thought, why isn't this a `label` prop in DpInput. But then, the headings before would also need to be labels and for the `Spaltennamen in DOCX-Exporten` this would not work out?

I think there is a semantic element which could be used for that:

```
<fieldset>
  <legend class="styled exactly like label">Spaltennamen in DOCX-Exporten</legend>
  <dp-input /> <dp-input /> <dp-input />
</fieldset>
```

However as this needs a little more refining, i just wanted to drop it here but no need to do it now.

Reply:
Thanks for pointing out. I'd like to look into that later, maybe next week, so the feature freeze can happen tomorrow (and I'm not working tomorrow).